app/forms/mobile_number_form.rb

```diff
@@ -48,4 +51,13 @@ def send_sms_message
   def mobile_number_changed?
     mobile_number != answers.mobile_number
   end
+
+  def handle_notify_error(error)
+    if error.message.include?("ValidationError: phone_number Number is not valid")
+      errors.add(:mobile_number, i18n_errors_path("invalid"))
+      false
```

Comment:
this `false` won't do anything? as there's a `false` after it in the caller

Reply:
Have removed it 👍